### ISSUE TITLE
Can't copy file because it cannot be found

### ISSUE THREAD

**Issue by tibuch** [fmi-basel/faim-robocopy#9]
[ERROR]: Robocopy returned with exit code 11. The following issues were encountered: [Code 2] Copying File path/to/file.stk: The system cannot find the file specified.

The file is available in the target directory and was most likely already copied by a previous process. However, this process started before the old process finished deleting the files. Hence, it ended up in this process queue but got deleted before it was reached.

Bottom line: No data is lost, but the error should be fixed.
